kasparovsyncd/sync/block.go

```diff
@@ -20,21 +20,24 @@ func insertBlocks(dbTx *database.TxContext, blocks []*rawAndVerboseBlock, transa
 			blockMass += transactionHashesToTxsWithMetadata[tx.Hash].mass
 		}
 		var err error
-		blocksToAdd[i], err = makeDBBlock(block.Verbose, blockMass)
+		blocksToAdd[i], err = dbBlockFromVerboseBlock(block.Verbose, blockMass)
 		if err != nil {
 			return err
 		}
 	}
 	return dbaccess.BulkInsert(dbTx, blocksToAdd)
 }
 
-func getBlocksAndParentIDs(dbTx *database.TxContext, blocks []*rawAndVerboseBlock) (map[string]uint64, error) {
+func getBlocksAcceptedAndParentIDs(dbTx *database.TxContext, blocks []*rawAndVerboseBlock) (map[string]uint64, error) {
```

Comment:
I think listing commas aren't really a thing in camel case.
Please rename this to something like getBlocksWithTheirAcceptedBlocksAndParentIDs.